CSS/index.css

```diff
@@ -1,4 +1,6 @@
-*{
+*,
+*::before,
+*::after{
```

Comment:
Added universal selectors to before and after psudo-elements to properly target them to remove padding, margin and include border box. Elements behave differently if they do not have the border-box property.